### PR TITLE
Tomer/add random partitioned option

### PR DESCRIPTION
**Pull request by tomerr-lb**

### CODE REVIEW

**Review by dori-lb — APPROVED** [review]

File: memtier_benchmark.cpp [hunk 1]
```diff
@@ -646,8 +649,8 @@ static int config_parse_args(int argc, char *argv[], struct benchmark_config *cf
                 case o_key_pattern:
                     cfg->key_pattern = optarg;
                     if (strlen(cfg->key_pattern) != 3 || cfg->key_pattern[1] != ':' ||
-                        (cfg->key_pattern[0] != 'R' && cfg->key_pattern[0] != 'S' && cfg->key_pattern[0] != 'G' && cfg->key_pattern[0] != 'P') ||
-                        (cfg->key_pattern[2] != 'R' && cfg->key_pattern[2] != 'S' && cfg->key_pattern[2] != 'G' && cfg->key_pattern[2] != 'P')) {
+                        (cfg->key_pattern[0] != 'C' && cfg->key_pattern[0] != 'R' && cfg->key_pattern[0] != 'S' && cfg->key_pattern[0] != 'G' && cfg->key_pattern[0] != 'P') ||
+                        (cfg->key_pattern[2] != 'C' && cfg->key_pattern[2] != 'R' && cfg->key_pattern[2] != 'S' && cfg->key_pattern[2] != 'G' && cfg->key_pattern[2] != 'P')) {
                             fprintf(stderr, "error: key-pattern must be in the format of [S/R/G]:[S/R/G].\n");
```
Comment (adir-lb) [inline memtier_benchmark.cpp]: don't you want to update the error message?

**Review by adir-lb — COMMENTED** [review]
seems legit

File: memtier_benchmark.cpp [hunk 2]
```diff
@@ -151,7 +151,6 @@ static void config_print(FILE *file, struct benchmark_config *cfg)
         cfg->data_verify ? "yes" : "no",
         cfg->verify_only ? "yes" : "no",
         cfg->verify_set_only ? "yes" : "no",
-        cfg->verify_set_only ? "yes" : "no",
```
Comment (sashas-lb) [inline memtier_benchmark.cpp:154]: did not you just add it couple of days ago?

File: client.cpp [hunk 3]
```diff
@@ -985,7 +985,8 @@ void crc_verify_client::handle_response(struct timeval timestamp, request *reque
         m_stats.update_get_latency_map(response->get_latency());
     }
 
-    if (response->is_error() || !values_count) {
+    if (strcmp(response->get_status(), "PROTOCOL_BINARY_RESPONSE_KEY_ENOENT") == 0 ||
+                                                response->is_error() || !values_count) {
```
Comment (sashas-lb) [inline client.cpp:989]: should not we also raise "failed to verify" counter?

File: memtier_benchmark.cpp (hunk 2 above)
Comment (tomerr-lb) [reply memtier_benchmark.cpp:154]: @sashas-lb  yes I asked you if you want me to just revert it from the master

File: client.cpp (hunk 3 above)
Comment (tomerr-lb) [reply client.cpp:989]: @sashas-lb the "failed to verified" stats is related to the errors count and this will increment the error counter

**Pull request merged**